### ISSUE TITLE
[Buildkite] Enable independent Elastic Agents and for some packages enable parallel system tests

### ISSUE THREAD

**Issue by mrodm** [elastic/integrations#10201]
Relates https://github.com/elastic/elastic-package/issues/787

Enable independent Elastic Agents for all packages, ~it will require to update `auditd_manager` and `oracle` packages.~ It needs to define the environment variable `ELASTIC_PACKAGE_TEST_ENABLE_INDEPENDENT_AGENT=true`
- ~Related to the changes in https://github.com/elastic/integrations/pull/9862~

Enable system tests in parallel for `network_traffic` and `zeek` packages:
- Requires to add the configuration file `_dev/test/config.yml`
- Requires to set the maximum number of routines with the environment variable `ELASTIC_PACKAGE_MAXIMUM_NUMBER_PARALLEL_TESTS=5`
- Related to the changes in https://github.com/elastic/integrations/pull/10161

## Checklist
- [x] Requires a new elastic-package version released and updated in this repository

EDIT: Move `auditd_manager` and `oracle` packages update to another issue.

**Comment by mrodm** [comment]
Updated description issue to just enable independent Elastic Agents and enable running system tests in parallel for `network_traffic` and `zeek` packages.

Updating `auditd_manager` and `oracle` packages is going to be done in a second stage, once `elastic-package` is running independent Elastic Agents by default. A new `elastic-package` release containing https://github.com/elastic/elastic-package/issues/1954 is required.

Created issue for this second stage: https://github.com/elastic/integrations/issues/10390
